app.coffee

```diff
@@ -55,9 +55,9 @@ mongoOptions =
 
 MongoClient.connect app.get('mongo url'), mongoOptions, (err, db) ->
   # Load up API
-  requireFu(__dirname + '/api/v1')(app, db)
+  #requireFu(__dirname + '/api/v1')(app, db)
 
   # Load up routes
-  #requireFu(__dirname + '/routes')(app, db)
+  requireFu(__dirname + '/routes')(app, db)
```

Comment:
Let's change this method to something like this:

```coffee
api = {}

requireFu(__dirname + '/api/v1')(api, db)
requireFu(__dirname + '/routes')(app, api, db)
```

That way *this file* is what is responsible for injecting dependencies into the other files. Note that with this change you'll have to change the number of arguments coming into files under `routes/`.